docs/appendices/release-notes/5.7.4.rst

```diff
@@ -48,6 +48,11 @@ See the :ref:`version_5.7.0` release notes for a full list of changes in the
 Fixes
 =====
 
+- Fixed an issue that would add a whitespace character at the beginning of some
+  lines in the files containing the rows which are exported by executing
+  :ref:`COPY TO<sql-copy-to>` using the local filesystem on a whole partitioned
```

Comment:
Is it specific to partitioned tables, or is that just where we saw it happening?  I think it would manifest any time COPY_TO FILE was run with json output on a shard with multiple rows.

Reply:
I've tested it with a simple table and multiple shards, and multiple docs per shard, but it's not happening there.

Reply:
That is because for a simple table we get one file per shard, whereas in the partitioned table case we get one file per partition, so when rows from a new shard are exported, the whitespace is added.